privacyidea/static/components/token/controllers/tokenDetailController.js

```diff
@@ -15,7 +15,7 @@ myApp.controller("tokenDetailController", function ($scope,
     $scope.selectedRealms = {};
     $scope.newUser = {user: "", realm: $scope.defaultRealm};
     $scope.loggedInUser = AuthFactory.getUser();
-    $scope.machinesPerPage = 5;
+    $scope.machinesPerPage = 15;
```

Comment:
Hm, we could make this configurable... but this might be an overkill. If there are only 2 machines attached, the list will only contain two hosts.
So I will merge it this way.

If you are interested in designing a better assignment including groups, please drop me a note.
See #285 
Any input is highly appreciated. Thanks for the patch.